providers/default.rb

```diff
@@ -143,6 +143,7 @@ def download_installer
 def run_installer(installer_cmd)
   execute 'Install Sumo Collector' do
     command "#{Chef::Config[:file_cache_path]}/#{installer_cmd}"
+    sensitive true
```

Comment:
While I 100% agree with the intent I think we need a way to enable people to debug without forking the cookbook or modifying it locally. What I have done in the past regarding this is to specify a debug flag that can be toggled in the case of failures and needing to see it on the console (not being sent to sumo).
```suggestion
    sensitive true if new_resource.debug == true
```

You will still need to add this to the resource as a parameter.

Reply:
see: https://github.com/sensu/sensu-go-chef/pull/44 for an example. I am gonna merge that in the other project once I can get the stupid chef license stuff figured out.